junit-platform-commons/src/main/java/org/junit/platform/commons/util/ExceptionUtils.java

```diff
@@ -31,6 +39,12 @@
 @API(status = INTERNAL, since = "1.0")
 public final class ExceptionUtils {
 
+	private static final String JUNIT_PLATFORM_LAUNCHER_PACKAGE_PREFIX = "org.junit.platform.launcher.";
+	private static final List<String> ALWAYS_INCLUDED_STACK_TRACE_ELEMENTS = Arrays.asList( //
+		"org.junit.jupiter.api.Assertions", //
+		"org.junit.jupiter.api.Assumptions" //
+	);
```

Comment:
Since these are Jupiter-specific, I think they should be passed in to `pruneStackTrace` or be checked in the already passed in `stackTraceElementFilter`.